bundles/org.openhab.binding.bloomsky/src/main/java/org/openhab/binding/bloomsky/internal/dto/BloomSkyJsonSensorData.java

```diff
@@ -0,0 +1,299 @@
+
+    @SerializedName("CityName")
+    @Expose
+    private String cityName; // City name description given to the BloomSky PWS by device owner
+
+    /*
+     * The Storm object sensor data if not null a Storm is associated
+     * with the device account.
+     */
+    @SerializedName("Storm")
+    @Expose
+    private BloomSkyStormData storm;
+
+    /*
+     * Searchable i.e. public can find this PWS on BloomSky map of weather stations,
+     * value is true or false.
+     */
+    @SerializedName("Searchable")
+    @Expose
+    private String searchable;
+
+    @SerializedName("DeviceName")
+    @Expose
+    private String deviceName; // Device name - Owner assigned name for the device
+
+    /*
+     * Registration time stamp - date and time when device was first registered with BloomSky account
+     * Epoch format (Unix time stamp)
+     */
+    @SerializedName("RegisterTime")
+    @Expose
+    private Long registerTime;
+
+    @SerializedName("DST")
+    @Expose
+    private Integer dST; // Daylight savings time on (1) or off (0)
+
+    @SerializedName("BoundedPoint")
+    @Expose
+    private String boundedPoint; // Bounded point - related to indoor devices no longer supported - should be null
+
+    @SerializedName("LON")
+    @Expose
+    private Double lON; // Longitude of the PWS
+
+    @SerializedName("Point")
+    @Expose
+    private BloomSkyPointData point; // Point - related to indoor devices no longer supported - should be null
+
+    /*
+     * Video list [Array]- URLs to last five days of video time lapse mp4's in Fahrenheit
+     */
+    @SerializedName("VideoList")
+    @Expose
+    private List<String> videoList = null;
+
+    /*
+     * Video list [Array]- URLs to last five days of video time lapse mp4's in Celsius
+     */
+    @SerializedName("VideoList_C")
+    @Expose
+    private List<String> videoListC = null;
+
+    @SerializedName("DeviceID")
+    @Expose
+    private String deviceID; // Device unique identifier
+
+    @SerializedName("NumOfFollowers")
+    @Expose
+    private Integer numOfFollowers; // Number of followers that have this PWS as a favorite
+
+    @SerializedName("LAT")
+    @Expose
+    private Double lAT; // Latitude of the PWS
+
+    @SerializedName("ALT")
+    @Expose
+    private Double aLT; // Altitude of the PWS
+
+    @SerializedName("Data")
+    @Expose
+    private BloomSkySkyData data; // Sensor Data for SKY1/2
+
+    @SerializedName("FullAddress")
+    @Expose
+    private String fullAddress; // Full location address
+
+    @SerializedName("StreetName")
+    @Expose
+    private String streetName; // Street name assigned by owner
+
+    /*
+     * Image snapshots list [Array]- URLs to last five image snapshots that make up the days video
+     */
+    @SerializedName("PreviewImageList")
+    @Expose
+    private List<String> previewImageList = null;
+
+    public Double getuTC() {
+        return uTC;
+    }
+
+    public void setuTC(Double uTC) {
```

Comment:
As your binding is read-only, not sure setters are from any interest.

Reply:
Removed these, they are not needed.